src/main/scala/org/graphframes/lib/AggregateMessages.scala

```diff
@@ -130,6 +130,13 @@ class AggregateMessages private[graphframes] (private val g: GraphFrame)
     }
     unionedMsgs.groupBy(ID).agg(aggCol)
   }
+
+  /**
+   * Run the aggregation, specifying SQL expression as a String
+   *
+   * See the overloaded method documentation for more details.
+   */
+  def agg(aggCol: String): DataFrame = agg(expr(aggCol))
```

Comment:
Would be great to add a Scala test for this - this helps mitigate the Codecov decrease too